src/ui/theme/components/button.ts

```diff
@@ -151,4 +152,15 @@ function variantToggle(props: any) {
   };
 }
 
+function variantHeaderNav() {
+  return {
+    py: 1,
+    border: 'none',
+    background: 'transparent',
```

Comment:
what is the difference between this and `bg` used in other variants?  Why use 'transparent' rather than white/sepia/dark ?

Reply:
Whoops, `bg` == `background` in chakraui, I will update it so it matches the rest of the code. 

We have a separate style for the toggle buttons and the rest of the elements.`white/sepia/dark` are the hardcoded colors specifically for the three buttons on the Settings Card, the HeaderNav background is dynamic and will change based on the color mode setting. (Not yet implemented)